apps/prairielearn/src/pages/instructorIssues/instructorIssues.html.ts

```diff
@@ -319,9 +319,8 @@ function IssueRow({
 
 function getFormattedMessage(issue: Issue) {
   if (!issue.student_message) return html`&mdash;`;
-
-  const message = joinHtml(issue.student_message.split(/\r?\n|\r/), html`<br />`);
-  return issue.manually_reported ? html`"${message}"` : message;
+  if (issue.manually_reported) return `"${issue.student_message}"`;
```

Comment:
```suggestion
  if (issue.manually_reported) return html`"${issue.student_message}"`;
```

Reply:
Why does this matter? The return value of this function gets immediately interpolated into another `html` tagged-template literal.

Reply:
The perils of reviewing it on the phone... You are right. I mainly compared it to the previous version and its returning type, but the new version is still valid based on the context of where the function is called.